File: trabalho 1/VitorBasso11611BCC034/programas tarefa 1/java/micro/micro07.java
public class micro07{
    public static void main(String[] args){
        int programa, numero, opc;

        programa = 1;
        while(programa == 1){
            System.out.print("Digite um numero: ");
            numero = Integer.parseInt(System.console().readLine());
            if(numero > 0){
                System.out.println("Positivo");
            }else{
                if(numero == 0){
                    System.out.println("O numero e igual a 0");
                }
                if(numero < 0){
                    System.out.println("Negativo");
                }
            }
            System.out.print("Deseja finalizar? (S -1) ");
            opc = Integer.parseInt(System.console().readLine());
            if(opc == 1){
                programa = 0;
            }

        }
    }
}

File: trabalho 4/codigos/lua/micro/micro07.lua
function int main()
    local int programa
    local int numero
    local int opc
    programa = 1
    while programa == 1 do
        print("Digite um numero: ")
        numero = io.read()
        if numero > 0 then
            print("Positivo\n")
        else
            if numero == 0 then
                print("O numero e igual a 0\n")
            end
            if numero < 0 then
                print("Negativo\n")
            end
        end
        print("Deseja finalizar? (S - 1): ")
        opc = io.read()
        if opc == 1 then
            programa = 0
        end
    end
end
